cloudferrylib/os/network/neutron.py

```diff
@@ -109,9 +110,19 @@ def deploy(self, info):
         if self.config.migrate.keep_floatingip:
             self.upload_floatingips(deploy_info['networks'],
                                     deploy_info['floating_ips'])
+            if self.config.migrate.change_router_ips:
+                subnets_map = {subnet['id']: subnet
+                           for subnet in deploy_info['subnets']}
+                router_ips = self.extract_router_ips_as_floating_ips(
+                    subnets_map, deploy_info['routers'])
+                dst_router_ip_ids = self.upload_floatingips(
+                    deploy_info['networks'], router_ips)
         self.upload_routers(deploy_info['networks'],
                             deploy_info['subnets'],
                             deploy_info['routers'])
+        if self.config.migrate.clean_router_ips_stub and dst_router_ip_ids:
```

Comment:
This option does not seem to have much value in real-world migration. Typical scenario I see:
1. Migrate resources (includes routers with new floating IPs)
2. Migrate VMs in multiple steps
If operator sets `clean_router_ips_stub` to `True`, new router IPs will be removed, which means internet connectivity during VM migration will be broken for migrated and non-migrated VMs.